Dockerfile

```diff
@@ -18,11 +41,13 @@ RUN addgroup --gid 11111 forgerock && \
 WORKDIR /opt/gen
 
 COPY --from=ds --chown=forgerock:root /opt/opendj /opt/gen/opendj
-COPY --from=build --chown=forgerock:root /go/src/github.com/ForgeRock/secret-agent/secret-agent /secret-agent
+COPY --from=builder --chown=forgerock:root /workspace/manager /
 
 RUN mkdir -p /opt/gen/secrets/generic/truststore && \
     cp $JAVA_HOME/lib/security/cacerts /opt/gen/secrets/generic/truststore && \
     chmod 764 /opt/gen/secrets/generic/truststore/cacerts && \
     chown -R forgerock:root /opt/gen
 
 USER forgerock
+ENTRYPOINT ["/manager"]
+
```

Comment:
Not everyone agrees with me on this, but over the 6+ years I've been working with Docker, I started setting ENTRYPOINT to bash in almost all my Dockerfiles. The reason is, in every instance where I am starting up the image manually, it's to debug something, so I _always_ end up adding `--entrypoint bash` to my `docker run` command. - And in every place where the image is started automatically for real usage, (whether it's a `docker-compose.yaml`, `deployment.yaml`, or something else,) I want to see the entrypoint explicitly set there so I can see the entire definition of what's being run all in one place. - E.G. env vars, entrypoint, command, volume mounts, - everything.

Having to go look in a Dockerfile when you're looking a `deployment.yaml` to find out what's actually being run is a drag. Having to override the entrypoint whenever you run the image manually is also a drag.